src/main/java/mate/academy/internetshop/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.internetshop.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.UserService;
+
+public class UserController extends HttpServlet {
```

Comment:
let's add verbs to Controller's names for example  GetAllUsersController (here and everywhere)